module/motion/KinematicsConfiguration/tests/KinematicsConfigurationTest.cpp

```diff
@@ -14,14 +14,71 @@
  * You should have received a copy of the GNU General Public License
  * along with the NUbots Codebase.  If not, see <http://www.gnu.org/licenses/>.
  *
- * Copyright 2015 NUbots <nubots@nubots.net>
+ * Copyright 2021 NUbots <nubots@nubots.net>
```

Comment:
Should we be changing these for all files we change from now on?

Reply:
I dunno. Given that this file was basically empty, it seems appropriate, but I have no idea about the legal stuff and licence things. I have just been doing when it seemed reasonable 😂

Reply:
Speaking of updating things, I'm pretty sure that email address doesn't work anymore.